parsers/cirrus.go

```diff
@@ -0,0 +1,43 @@
+package parsers
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/cirruslabs/cirrus-ci-annotations/model"
+	"os"
+)
+
+const currentReportVersion = "1"
+
+type cirrusReport struct {
```

Comment:
What do you think of having a file containing `model.Annotation` separated by new lines instead of a single object? I think this way a tool can easily stream annotations to a file instead of accumulating them all first?

Not sure how to do about `version` though... Maybe just include it in the annotation? Or don't introduce it until we have version `2` and then respect it but default to `1` in case it's missing.

Reply:
Worth a try, implemented in https://github.com/cirruslabs/cirrus-ci-annotations/pull/34/commits/d0adbc7065dc0dff273c7395c566539442a8f76c.